src/Exports.jl

```diff
@@ -1151,6 +1151,7 @@ export simulate_valuation
 export singular
 export singular_assure
 export singular_coeff_ring
+export singular_ideal
```

Comment:
I don't think this should be exported, but for internal use only. No user of Oscar should ever handle Singular ideals.

Reply:
Yes, all internal.